Compilation error with bit-set 0.5.0

Using proptest 1.3.0 and bit-set 0.5.0 with the following Cargo.toml results in a compilation error.

```toml
[dependencies]
proptest = "1.3.0"
bit-set = "=0.5.0"
```

```
   Compiling proptest v1.3.0
error[E0308]: mismatched types
   --> C:\Users\frozenlib\.cargo\registry\src\index.crates.io-6f17d22bba15001f\proptest-1.3.0\src\bits.rs:463:39
    |
463 |             Self(BitSet::from_bit_vec(BitVec::from_elem(len, true)))
    |                  -------------------- ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `bit_vec::BitVec`, found a different `bit_vec::BitVec`
    |                  |
    |                  arguments to this function are incorrect
    |
    = note: `bit_vec::BitVec` and `bit_vec::BitVec` have similar names, but are actually distinct types
note: `bit_vec::BitVec` is defined in crate `bit_vec`
   --> C:\Users\frozenlib\.cargo\registry\src\index.crates.io-6f17d22bba15001f\bit-vec-0.6.3\src\lib.rs:212:1
    |
212 | pub struct BitVec<B=u32> {
    | ^^^^^^^^^^^^^^^^^^^^^^^^
note: `bit_vec::BitVec` is defined in crate `bit_vec`
   --> C:\Users\frozenlib\.cargo\registry\src\index.crates.io-6f17d22bba15001f\bit-vec-0.5.1\src\lib.rs:215:1
    |
215 | pub struct BitVec<B=u32> {
    | ^^^^^^^^^^^^^^^^^^^^^^^^
    = note: perhaps two different versions of crate `bit_vec` are being used?
note: associated function defined here
   --> C:\Users\frozenlib\.cargo\registry\src\index.crates.io-6f17d22bba15001f\bit-set-0.5.0\src\lib.rs:220:12
    |
220 |     pub fn from_bit_vec(bit_vec: BitVec) -> Self {
    |            ^^^^^^^^^^^^
```

[bit-set was changed to use bit-vec 0.6 instead of 0.5 in bit-set version 0.5.2](https://github.com/contain-rs/bit-set/commit/90ad34c4cb15d15c310f5926c9e5d1b76617e699), so I believe proptest, which uses bit-vec 0.6, needs to depend on bit-set 0.5.2 or later, not 0.5.0.

## Version
cargo 1.72.1 (103a7ff2e 2023-08-15)
proptest 1.3.0

--- a/proptest/CHANGELOG.md
+++ b/proptest/CHANGELOG.md
@@ -1,5 +1,11 @@
 ## Unreleased
 
+## 1.3.1
+
+## Other Notes
+
+- `bit-set` updated from 0.5.0 to 0.5.2 to ensure minimum compatible version with bit-vec 0.6
+
 ## 1.3.0
 
 ### Breaking Changes

--- a/proptest/Cargo.toml
+++ b/proptest/Cargo.toml
@@ -79,7 +79,7 @@ version = "0.7"
 optional = true
 
 [dependencies.bit-set]
-version = "0.5.0"
+version = "0.5.2"
 optional = true
 
 [dependencies.bit-vec]

--- a/proptest/Cargo.toml
+++ b/proptest/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "proptest"
-version = "1.3.0"
+version = "1.3.1"
 authors = ["Jason Lingle"]
 license = "MIT OR Apache-2.0"
 readme = "README.md"

